application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue

```diff
@@ -257,6 +257,7 @@ export default {
       confirmOKMessage: '',
       confirmKOMessage: '',
       confirmAction(){return;},
+      downloadEnabled: true,
```

Comment:
```suggestion
      downloadEnabled: false,
```
Please make sure to enable download only when it's permitted.